lib/cogctl/action_util.ex

```diff
@@ -47,12 +73,20 @@ defmodule Cogctl.ActionUtil do
       {:ok, endpoint_with_token} ->
         fun.(endpoint_with_token)
       {:error, error} ->
+        error = case error do
+                  error when is_list(error) ->
+                    Enum.join(error, "\n")
+                  error ->
+                    inspect(error)
+                end
+
```

Comment:
I got rid of this entirely and built a `format_error` function with a couple of different heads. Reads way better now.